jobs/src/test/scala/dcos/metronome/jobrun/impl/JobRunExecutorActorTest.scala

```diff
@@ -320,7 +320,7 @@ class JobRunExecutorActorTest extends TestKit(ActorSystem("test"))
   test("Init of JobRun with JobRunStatus.Success") {
     val f = new Fixture
     import f._
-    val successfulJobRun = JobRun(JobRunId(defaultJobSpec), defaultJobSpec, JobRunStatus.Success, clock.instant(), None, None, Map.empty)
+    val successfulJobRun = JobRun(JobRunId(defaultJobSpec), defaultJobSpec, JobRunSpecOverrides.empty, JobRunStatus.Success, clock.instant(), None, None, Map.empty)
```

Comment:
would be nice to add tests here to see that if I specify override, job (RunSpec) with that overrides applied is actually created

Reply:
Added!